spin/cmds/meson.py

```diff
@@ -257,7 +278,19 @@ def build(
     or set CFLAGS appropriately:
 
       CFLAGS="-O0 -g" spin build
+
+    Build into a different build/build-install directory:
+
+      spin build -C build-for-feature-x
+
+    This feature is useful in combination with a shell alias, e.g.:
+
+      $ alias spin-clang="spin -C build-clang"
```

Comment:
`spin -C ${SOME_DIR} command` seems to fail. I think this is because the option is added ad subcommand level. So this alias won't work currently. I think the option would need to be added to the top-level `spin` command and then passed down via the [context](https://click.palletsprojects.com/en/8.1.x/complex/#contexts).

Reply:
Right, may have to rethink that. We don't have global context in spin.